### ISSUE TITLE
Consensus jobs failed, tried 2 times, giving up

### ISSUE THREAD

**Issue by vaee4** [marbl/canu#1978]
Hi,
I am getting errors while running my nanopre fastq file on canu version 2.1.1 on Linux. 
I installed it by downloading the binary release.

```
-- Running jobs.  First attempt out of 2.
----------------------------------------
-- Starting 'cns' concurrent execution on Thu Jul  8 20:12:43 2021 with 45.622 GB free disk space (15 processes; 2 concurrently)

    cd unitigging/5-consensus
    ./consensus.sh 1 > ./consensus.000001.out 2>&1
    ./consensus.sh 2 > ./consensus.000002.out 2>&1
    ./consensus.sh 3 > ./consensus.000003.out 2>&1
    ./consensus.sh 4 > ./consensus.000004.out 2>&1
    ./consensus.sh 5 > ./consensus.000005.out 2>&1
    ./consensus.sh 6 > ./consensus.000006.out 2>&1
    ./consensus.sh 7 > ./consensus.000007.out 2>&1
    ./consensus.sh 8 > ./consensus.000008.out 2>&1
    ./consensus.sh 9 > ./consensus.000009.out 2>&1
    ./consensus.sh 10 > ./consensus.000010.out 2>&1
    ./consensus.sh 11 > ./consensus.000011.out 2>&1
    ./consensus.sh 12 > ./consensus.000012.out 2>&1
    ./consensus.sh 13 > ./consensus.000013.out 2>&1
    ./consensus.sh 14 > ./consensus.000014.out 2>&1
    ./consensus.sh 15 > ./consensus.000015.out 2>&1

-- Finished on Thu Jul  8 20:12:48 2021 (5 seconds) with 45.617 GB free disk space
----------------------------------------
-- Using slow alignment for consensus (iteration '1').
-- Configured 15 consensus jobs.
--
-- Consensus jobs failed, retry.
--   job ctgcns/0002.cns FAILED.
--   job ctgcns/0006.cns FAILED.
--   job ctgcns/0011.cns FAILED.
--
--
-- Running jobs.  Second attempt out of 2.
----------------------------------------
-- Starting 'cns' concurrent execution on Thu Jul  8 20:12:48 2021 with 45.617 GB free disk space (3 processes; 2 concurrently)

    cd unitigging/5-consensus
    ./consensus.sh 2 > ./consensus.000002.out 2>&1
    ./consensus.sh 6 > ./consensus.000006.out 2>&1
    ./consensus.sh 11 > ./consensus.000011.out 2>&1

-- Finished on Thu Jul  8 20:12:48 2021 (furiously fast) with 45.617 GB free disk space
----------------------------------------
-- Using slow alignment for consensus (iteration '2').
-- Configured 15 consensus jobs.
--
-- Consensus jobs failed, tried 2 times, giving up.
--   job ctgcns/0002.cns FAILED.
--   job ctgcns/0006.cns FAILED.
--   job ctgcns/0011.cns FAILED.
--

ABORT:
ABORT: canu 2.1.1
ABORT: Don't panic, but a mostly harmless error occurred and Canu stopped.
ABORT: Try restarting.  If that doesn't work, ask for help.
ABORT:

```

I am pasting the error message from of the consensus file (consensus.000002.out); 
`Failed with 'Segmentation fault'; backtrace (libbacktrace):`

I initially assembled the fastq file without barcode trimming and it worked well, but the same sample after barcode trimming gives this error message.
I have attached the complete report of the run and the consensus.000002.out for your reference.
Can you please help to fix this issue?
Thank you
[consensus.000002.out.odt](https://github.com/marbl/canu/files/6783972/consensus.000002.out.odt)
[canu-vr.odt](https://github.com/marbl/canu/files/6783973/canu-vr.odt)

**Comment by skoren** [comment]
This is likely the same as #1964 (see also #1901 and #1864). The fix is to update the source as listed in #1964 and recompile the distribution. If you cannot do that, I've also attached an updated binary (md5:9b3846d82c15bbab37536838ae4e640e after uncompressing) for linux to this issue. You can uncompress it and replace the one in your canu installation and try running one of the failed jobs again to see if they succeed.
[utgcns.gz](https://github.com/marbl/canu/files/6786262/utgcns.gz)

**Comment by vaee4** [comment]
Thank you vey much for your prompt response.
I dowloaded utgcns.gz and extracted and compiled in bin and it worked right away.
Thank you for your help.
